htdocs/index.php

```diff
@@ -437,7 +437,7 @@
 
 	// We calculate $totallate. Must be defined before start of next loop because it is show in first fetch on next loop
 	foreach ($valid_dashboardlines as $board) {
-		if ($board->nbtodolate > 0) {
+		if ($board->nbtodolate > 0 && is_numeric($board->nbtodo) && is_numeric($board->nbtodolate)) {
```

Comment:
So you should move the test $board->nbtodolate > 0 after the test is_numeric($board->nbtodolate) to avoid to make the comparison if value is not numeric